collect_datasets.py

```diff
@@ -1,14 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
this file is renamed and changed to `create_extension_file.py` but the diff doesn't seem to pick this up.